src/actions/toaster.js

```diff
@@ -0,0 +1,33 @@
+import actionTypes from '../constants/actions';
+
+/**
+ * An action to dispatch to display a toast
+ *
+ */
+export const toastDisplayed = data => ({
+  data,
+  type: actionTypes.toastDisplayed,
+});
+
+/**
+ * An action to dispatch to display a success toast
+ *
+ */
+export const successToastDisplayed = ({ type = 'success', ...rest }) =>
+  toastDisplayed({ type, ...rest });
+
+
+/**
+ * An action to dispatch to display an error toast
+ *
+ */
+export const errorToastDisplayed = ({ type = 'error', ...rest }) =>
+  toastDisplayed({ type, ...rest });
+
+/**
+ * An action to dispatch to hide a toast
+ *
+ */
+export const toastHidden = () => ({
```

Comment:
This doesn't handle multiple toast. we're using toasts to notify user about many changes, like sign message and network status changes. There's a good chance we'll need to show multiple - 2 is more realistic - toasts at the same time.
Please change this to include `id` and tweak other parts of code accordingly.
thanks